replacement_scan.go

```diff
@@ -0,0 +1,65 @@
+package duckdb
+
+/*
+   #include <stdlib.h>
+   #include <duckdb.h>
+   void replacement_scan_cb(duckdb_replacement_scan_info info, const char *table_name, void *data);
+   typedef const char cchar_t;
```

Comment:
Using `*C.char` directly does not work?

Reply:
Yeah, I remember this was a bit tricky. Perhaps there are better alternatives.

Reply:
Okay, no worries.